eng/common/scripts/get-codeowners.ps1

```diff
@@ -0,0 +1,43 @@
+param (
+  $TargetDirectory, # should be in relative form from root of repo. EG: sdk/servicebus
+  $RootDirectory # ideally $(Build.SourcesDirectory)
+)
+
+$codeOwnersLocation = @(Get-ChildItem -R -Force -Path $RootDirectory -Filter "CODEOWNERS")
```

Comment:
In order for github to find the CODEOWNERS files it must be in the `.github` folder so I think it is safe to just compute that path and test that it exists.

Reply:
Ah. Fair enough.